addons/resource/models/resource.py

```diff
@@ -151,29 +151,28 @@ def default_get(self, fields):
         res = super(ResourceCalendar, self).default_get(fields)
         if not res.get('name') and res.get('company_id'):
             res['name'] = _('Working Hours of %s') % self.env['res.company'].browse(res['company_id']).name
+        if not res.get('attendance_ids'):
```

Comment:
Could you explain why you had to move this inside the default_get?

Reply:
It's useful for resource in payroll, we override the default_get, inside we copy default attendance_ids to normal_attendance_ids and remove default attendance_ids to avoid to have default data 2 times